app/webpacker/src/stylesheets/_measure-inset.scss

```diff
@@ -27,6 +27,10 @@
     li:not(:last-child) {
       margin-bottom: 5px;
     }
+
+    ul {
+      padding-left: 27px;
```

Comment:
Not  sure how specific this sizing needs to be but if it can flex by a couple of pixels you could use whats provided by the design system since they have padding classes for 25px?

Reply:
25px doesn't align properly so I have to stick with 27px